tsfresh/examples/robot_execution_failures.py

```diff
@@ -102,10 +102,7 @@ def load_robot_execution_failures():
             if line[0] not in ['\t', '\n']:
                 cur_id += 1
                 time = 0
-                if line.strip() == 'normal':
-                    id_to_target[cur_id] = 0
-                else:
-                    id_to_target[cur_id] = 1
+                id_to_target[cur_id] = line.strip()
```

Comment:
can you add a Boolean flag that controls if a binary or a multi classification target is returned?